app/experimenter/static/core/package.json

```diff
@@ -91,13 +92,22 @@
         "@babel/plugin-proposal-decorators",
         {
           "legacy": true
-        }
+        },
+        "@babel/plugin-proposal-decorators"
       ],
       [
         "@babel/plugin-proposal-class-properties",
         {
           "loose": true
-        }
+        },
+        "@babel/plugin-proposal-class-properties"
+      ],
+      [
+        "@babel/plugin-proposal-private-methods",
+        {
+          "loose": true
+        },
+        "@babel/plugin-proposal-private-methods"
```

Comment:
Added this because I kept seeing this error when building `core`:

```
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-methods since the "loose" mode option was set to "true" for @babel/plugin-proposal-class-properties.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-methods", { "loose": true }]
```